packages/js/product-editor/src/components/header/preview-button/preview-button.tsx

```diff
@@ -23,6 +25,9 @@ export function PreviewButton( {
 	const previewButtonProps = usePreview( {
 		productStatus,
 		...props,
+		onClick() {
+			recordEvent( 'product_preview_changes', { source: TRACKS_SOURCE } );
```

Comment:
I'm wondering if we want this `source` added to all events and if so, whether or not we should create a new `recordProductEvent` (or enhance the current one we have) that wraps this and helps adding the source.

Reply:
Also, I'm wondering if the `source` property should be `_source` to help prevent potential conflicts.

Reply:
I like the idea of adding the source directly into the `recordProductEvent`. Renaming `source` to `_source` makes a lot of sense as well.